Formula/lib/libsail.rb

```diff
@@ -0,0 +1,59 @@
+class Libsail < Formula
+  desc "Missing small and fast image decoding library for humans (not for machines)"
+  homepage "https://github.com/HappySeaFox/sail"
+  url "https://github.com/HappySeaFox/sail/archive/refs/tags/v0.9.0.tar.gz"
+  sha256 "892738e0f56fed8c6387e1045bba2bfbf1b095024a495845d4879edb310cd1a7"
+  license "MIT"
+
+  depends_on "cmake"      => :build
+  depends_on "pkg-config" => [:build, :test]
+
+  depends_on "giflib"
+  depends_on "jasper"
+  depends_on "jpeg-turbo"
+  depends_on "jpeg-xl"
+  depends_on "libavif"
+  depends_on "libpng"
+  depends_on "libtiff"
+  depends_on "resvg"
+  depends_on "webp"
+
+  def install
+    args = %W[
+      -DCMAKE_INSTALL_RPATH=#{rpath}
+      -DBUILD_TESTING=ON
+      -DSAIL_BUILD_EXAMPLES=OFF
+    ]
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, *args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+    system "ctest", "--test-dir", "build/tests", "--rerun-failed", "--output-on-failure"
```

Comment:
no need to run tests per the release build, it should be part of the upstream pipeline already

```suggestion
```